tsfile/src/main/java/org/apache/iotdb/tsfile/read/TsFileSequenceReader.java

```diff
@@ -167,8 +167,8 @@ public void loadMetadataSize() throws IOException {
     fileMetadataSize = ReadWriteIOUtils.readInt(metadataSize);
     fileMetadataPos =
         tsFileInput.size() - TSFileConfig.MAGIC_STRING.length() - Integer.BYTES - fileMetadataSize;
-    // skip the magic header
-    tsFileInput.position(TSFileConfig.MAGIC_STRING.length());
+    // skip the magic header and version number
+    tsFileInput.position(TSFileConfig.MAGIC_STRING.length() + TSFileConfig.VERSION_NUMBER.length());
```

Comment:
But why `TSFileConfig.MAGIC_STRING` uses `String.length()`? It should be changed together? `position()` in here isn't called by me, it used to be. I think `position()` is used extensively in other places, not just here.

Reply:
I guess that is a historical problem or bug I am afraid.
Yes, `position()` is common elsewhere, because these are provided with a position and calling `position()` is necessary. But I cannot come up with the point of calling it here.

Reply:
Yes, I also think call `position()` in here is unnecessary. If remove `position()` in here, some tests has `readMarker()` should be also removed or changed. But what about `TSFileConfig.MAGIC_STRING`? It should use `String.getBytes().length()`?

Reply:
I think `String.getBytes().length()` should be used and I think if you add corresponding tests, you may be able to expose this bug.

Reply:
@jixuan1989 do you have some suggestions about that?